deploy/helm/charts/README.md

```diff
@@ -138,55 +138,113 @@ The following table lists the configurable parameters of the OpenEBS CStor chart
 | cstorCSIPlugin.image.registry | string | `nil` | CStor CSI driver image registry |
 | cstorCSIPlugin.image.repository | string | `"openebs/cstor-csi-driver"` |  CStor CSI driver image repository |
-| cstorCSIPlugin.image.tag | string | `"2.6.0"` | CStor CSI driver image tag |
+| cstorCSIPlugin.image.tag | string | `"2.7.0"` | CStor CSI driver image tag |
 | cstorCSIPlugin.name | string | `"cstor-csi-plugin"` | CStor CSI driver container name |
 | cstorCSIPlugin.remount | string | `"true"` | Enable/disable auto-remount when volume recovers from read-only state |
 | cvcOperator.annotations | object | `{}` | CVC operator annotations |
 | cvcOperator.componentName | string | `"cvc-operator"` | CVC operator component name |
 | cvcOperator.image.pullPolicy | string | `"IfNotPresent"` | CVC operator image pull policy  |
 | cvcOperator.image.registry | string | `nil` | CVC operator image registry |
 | cvcOperator.image.repository | string | `"openebs/cvc-operator"` | CVC operator image repository |
-| cvcOperator.image.tag | string | `"2.6.0"` | CVC operator image tag |
+| cvcOperator.image.tag | string | `"2.7.0"` | CVC operator image tag |
 | cvcOperator.nodeSelector | object | `{}` | CVC operator pod nodeSelector |
 | cvcOperator.podAnnotations | object | `{}` | CVC operator pod annotations |
 | cvcOperator.resources | object | `{}` |CVC operator pod resources  |
 | cvcOperator.resyncInterval | string | `"30"` | CVC operator resync interval |
 | cvcOperator.securityContext | object | `{}` | CVC operator security context |
 | cvcOperator.target.image.registry | string | `nil` | Volume Target image registry  |
 | cvcOperator.target.image.repository | string | `"openebs/cstor-istgt"` | Volume Target image repository |
-| cvcOperator.target.image.tag | string | `"2.6.0"` | Volume Target image tag |
+| cvcOperator.target.image.tag | string | `"2.7.0"` | Volume Target image tag |
 | cvcOperator.tolerations | list | `[]` | CVC operator pod tolerations |
 | cvcOperator.volumeExporter.image.registry | string | `nil` | Volume exporter image registry |
 | cvcOperator.volumeExporter.image.repository | string | `"openebs/m-exporter"` | Volume exporter image repository |
-| cvcOperator.volumeExporter.image.tag | string | `"2.6.0"` | Volume exporter image tag |
+| cvcOperator.volumeExporter.image.tag | string | `"2.7.0"` | Volume exporter image tag |
 | cvcOperator.volumeMgmt.image.registry | string | `nil` | Volume mgmt image registry |
 | cvcOperator.volumeMgmt.image.repository | string | `"openebs/cstor-volume-manager"` | Volume mgmt image repository |
-| cvcOperator.volumeMgmt.image.tag | string | `"2.6.0"` |  Volume mgmt image tag|
+| cvcOperator.volumeMgmt.image.tag | string | `"2.7.0"` |  Volume mgmt image tag|
 | imagePullSecrets | string | `nil` | Image registry pull secrets |
 | openebsNDM.enabled | bool | `true` | Enable OpenEBS NDM dependency |
+| openebs-ndm.featureGates.APIService.enabled | bool | `true` | Enable 'API Service' feature gate for NDM |
```

Comment:
Also, its not required to add this here, as all the variables are documented in the dependent NDM chart

Reply:
Added this based on https://github.com/openebs/cstor-operators/issues/262

Reply:
That feature gate is user configurable, not a default setting.

Reply:
Removed all ndm attributes except image registry/repository, feature gates, filters, ndm nodeselector.